app/assets/src/components/common/ProjectCreationForm.jsx

```diff
@@ -9,7 +9,7 @@ import cs from "./project_creation_form.scss";
 class ProjectCreationForm extends React.Component {
   state = {
     name: "",
-    publicAccess: 1,
+    publicAccess: -1, // No selection by default
```

Comment:
I would disable the button until the user picks one of the options.

Reply:
👍